README.md

```diff
@@ -64,4 +64,6 @@ com.sun.tools.attach.AttachNotSupportedException: Unable to open socket file: ta
 	at Log4jHotPatch.loadInstrumentationAgent(Log4jHotPatch.java:182)
 	at Log4jHotPatch.main(Log4jHotPatch.java:259)
 ```
-this means you're running as a different user (including root) than the target JVM. JDK 8 can't handle patching as root user (and triggers a thread dump in the target JVM which is harmless). In JDK 11 patching a non-root process from a root process works just fine.
+this means you're running as a different user (including root) than the target JVM. JDK 8 can't handle patching as root user (and triggers a thread dump in the target JVM which is harmless). In JDK 11 patching a non-root process from a root process works just fine. 
+
+**Important:** If you attempted to patch as the wrong user, you may need to delete `.attach_pid<pid>` files (found in `/tmp` and/or the CWD of the VM process) before trying again. These files need to have the right ownership for attach to succeed.
```

Comment:
Can you expand on this? What was the error you saw? What JDK versions were you running for this?

Reply:
I first ran the hotpatch, but used the wrong UID (root).
Then ran the hotpatcher again, this time under the correct UID, and still got the error above:

~~~
com.sun.tools.attach.AttachNotSupportedException: Unable to open socket file: target process not responding or HotSpot VM not loaded
~~~

I had to switch back to root and delete the `.attach_pid<pid>` files in order for the hotpatcher to create them again (with the correct ownership).

My explanation for this is:

The attach mechanism works by creating `.attach_pid<pid>` files in CWD (or /tmp) and then sending SIGQUIT to the VM process. Under JDK8, the VM process will apparently ignore the request unless the file's ownership matches the process UID. When running the hotpatcher as the correct UID, it has no permission to delete the current `.attach_pid<pid>` (which were previously created as root), so the VM doesn't start the listener thread.

Reply:
Thank you for this patch and the explanation.